openapi3filter/options.go

```diff
@@ -7,6 +7,9 @@ type Options struct {
 	// Set ExcludeRequestBody so ValidateRequest skips request body validation
 	ExcludeRequestBody bool
 
+	// Set ExcludeRequestQuery so ValidateRequest skips request query params validation
+	ExcludeRequestQuery bool
```

Comment:
LGTM. yup add a test that makes sure the flag works
I'd also prefer a more precise name such as `ExcludeRequestQueryParams`

Reply:
Ok cool, i'll update the PR then